core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalH2OBackend.scala

```diff
@@ -170,8 +170,8 @@ object InternalH2OBackend extends InternalBackendUtils {
     }
   }
 
-  private def initializeH2OHiveSupport(conf: H2OConf, user: String): Unit = {
-    if (conf.isHiveSupportEnabled) {
+  private def initializeH2OKerberizedHiveSupport(conf: H2OConf, user: String): Unit = {
```

Comment:
@mn-mikke @honzasterba one question unrelated to this change. The user is never used within this method, isn't that a bug?

Reply:
this might be a remnant of the times when the keytab was used here to impersonate and refresh the token, it can be removed now I think

Reply:
yup, can be removed

Reply:
👍 thanks!